app/models/execute_twiml.rb

```diff
@@ -223,6 +226,17 @@ def execute_dial(verb)
     )
   end
 
+  def execute_record(verb)
+    attributes = twiml_attributes(verb)
```

Comment:
We need to translate TwiML Record's verb attributes into Adhearsion record's parameters.